### ISSUE TITLE
Problem installing with pip

### ISSUE THREAD

**Issue by ArsenicBismuth** [wqking/obfupy#1]
Hi, wanted to try your code but got into this issue while using `pip install obfupy` using Python 3.10 and pip 24.0:

``` python
Collecting obfupy
  Using cached obfupy-0.1.0.tar.gz (28 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [29 lines of output]
      running egg_info
      creating C:\Users\USER\AppData\Local\Temp\pip-pip-egg-info-8xcodenv\obfupy.egg-info
      writing C:\Users\USER\AppData\Local\Temp\pip-pip-egg-info-8xcodenv\obfupy.egg-info\PKG-INFO
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "C:\Users\USER\AppData\Local\Temp\pip-install-_tn23t1m\obfupy_c0f5e67bda664d31bfba6579a6796291\setup.py", line 15, in <module>
          setuptools.setup(
        File "D:\ProgramFiles\Miniconda3\envs\maya\lib\site-packages\setuptools\__init__.py", line 108, in setup
          return distutils.core.setup(**attrs)
        File "D:\ProgramFiles\Miniconda3\envs\maya\lib\site-packages\setuptools\_distutils\core.py", line 184, in setup
          return run_commands(dist)
        File "D:\ProgramFiles\Miniconda3\envs\maya\lib\site-packages\setuptools\_distutils\core.py", line 200, in run_commands
          dist.run_commands()
        File "D:\ProgramFiles\Miniconda3\envs\maya\lib\site-packages\setuptools\_distutils\dist.py", line 970, in run_commands
          self.run_command(cmd)
        File "D:\ProgramFiles\Miniconda3\envs\maya\lib\site-packages\setuptools\dist.py", line 945, in run_command
          super().run_command(command)
        File "D:\ProgramFiles\Miniconda3\envs\maya\lib\site-packages\setuptools\_distutils\dist.py", line 989, in run_command
          cmd_obj.run()
        File "D:\ProgramFiles\Miniconda3\envs\maya\lib\site-packages\setuptools\command\egg_info.py", line 303, in run
          writer(self, ep.name, os.path.join(self.egg_info, ep.name))
        File "D:\ProgramFiles\Miniconda3\envs\maya\lib\site-packages\setuptools\command\egg_info.py", line 666, in write_pkg_info
          metadata.write_pkg_info(cmd.egg_info)
        File "D:\ProgramFiles\Miniconda3\envs\maya\lib\site-packages\setuptools\_core_metadata.py", line 137, in write_pkg_info
          self.write_pkg_file(f)
        File "D:\ProgramFiles\Miniconda3\envs\maya\lib\site-packages\setuptools\_core_metadata.py", line 212, in write_pkg_file
          if not long_description.endswith("\n"):
      AttributeError: 'tuple' object has no attribute 'endswith'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by wqking** [comment]
Hi thanks for the issue report, I will check it when I'm on computer. I guess it's caused by miniconda you use.  
If you only want to see how it works, you don't need to install it, you can use the [scaffolding script](https://github.com/wqking/obfupy/blob/master/doc/scaffolding.md)

**Comment by ArsenicBismuth** [comment]
Nah, I wanted to actually implement this if it's good for our usecase. Ok thanks.

**Comment by wqking** [comment]
I fixed the problem and release a new version 0.1.1, you may try it again and let me know if it works. I tested it with miniconda and it works.
There is a typo in the setup script which makes the summary field a `list` instead of a `string`, and miniconda doesn't accept the list.
